app/screens/onboarding/index.tsx

```diff
@@ -111,6 +112,19 @@ const Onboarding = ({
         };
     }, []);
 
+    useEffect(() => {
```

Comment:
Any reason not to use the hook here?

Reply:
Yes, the hook execute a callback and returns true as it was handled, in this case we want to also return false in the event it wasn't